enigmatry-entry-blueprint-app/Enigmatry.Entry.Blueprint.App.csproj

```diff
@@ -1,55 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
-    <TypeScriptToolsVersion>4.4</TypeScriptToolsVersion>
-    <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
```

Comment:
Are you certain that this does not causes problems. I have added this a long time ago because Visual Studio was compiliing TS files and reporting various errors.